src/main/java/com/mlaf/hu/decisionagent/behavior/UpdateBehavior.java

```diff
@@ -0,0 +1,34 @@
+package com.mlaf.hu.decisionagent.behavior;
+
+import com.mlaf.hu.decisionagent.DecisionAgent;
+import com.mlaf.hu.models.InstructionSet;
+import jade.core.behaviours.CyclicBehaviour;
+import jade.lang.acl.ACLMessage;
+import jade.lang.acl.MessageTemplate;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+
+public class UpdateBehavior extends CyclicBehaviour {
```

Comment:
Misschien een beter beschrijvende naam geven zoals "UpdateActiveSensorAgentBehavior" ofzo?

Reply:
Ja is niet geweldig. Misschien CheckSensorAgentStatusBehavior?